includes/utils.inc

```diff
@@ -205,6 +205,9 @@ function wf_crm_get_events($reg_options, $context) {
   if (is_numeric($reg_options['event_type'])) {
     $sql .= ' AND event_type_id = ' . $reg_options['event_type'];
   }
+  if (is_numeric($reg_options['show_public_events'])) {
+    $sql .= ' AND is_public = ' . $reg_options['show_public_events'];
+  }
```

Comment:
For backward compatibility with existing forms in which this variable won't be set, I think the conditional should be

    if (is_numeric(wf_crm_aval($reg_options, 'show_public_events'))) {

Reply:
Updated as suggested